### PR TITLE
Surrender form endpoint

### PR DESCRIPTION
**Pull request by xshigeox**
Added the sever post endpoint to persist the data from the surrender form to the database.  Updated the link in the navbar to render the surrender form.

### CODE REVIEW

File: public/style.css [hunk 1]
```diff
@@ -163,6 +168,8 @@
   color: #fefefe;
   text-shadow: 1px 1px 2px #0a0a0a;
 }
+<<<<<<< HEAD
+=======
```
Comment (prophetsfall) [inline public/style.css]: Please make sure to resolve all merge conflicts prior to submission, and that all merge markers are removed.

File: server/server.js [hunk 2]
```diff
@@ -44,10 +44,10 @@ app.get("/api/v1/pet_type", (req, res)  =>  {
     })
 })
 
-app.get("/api/v1/:pet_type", (req, res)  =>  {
+app.get("/api/v1/:pet_type", (req, res) => {
   let petType = req.params.pet_type
-  if(petType == 'pigguinea' || petType == 'guineapig')  {
-    petType = 'guinea'
+  if (petType == "pigguinea" || petType == "guineapig") {
```
Comment (prophetsfall) [inline server/server.js]: I'm curious of the use case for getting "pigguinea" in your fetch. Can you clarify?

File: src/components/App.js [hunk 3]
```diff
@@ -11,7 +11,7 @@ const App = props => {
       </BrowserRouter>
       <Orbit />
     </div>
-  );
-};
+  )
+}
 
-export default App;
+export default App
```
Comment (prophetsfall) [inline src/components/App.js]: y'all've been doing the semi-colon dance a lot. 
Please decide as a group to not use semi-colons, and it'll save some confusion.

File: src/components/SurrenderForm.js [hunk 4]
```diff
@@ -48,11 +51,25 @@ const SurrenderForm = props => {
...
-      setNewSurrender(defaultFormValues)
+      fetch("/api/v1/newPet", {
+        method: "POST",
+        body: JSON.stringify(newSurrender),
+        headers: { "Content-Type": "application/json" }
+      })
+        .then(response => {
+          console.log(response)
+          if (response.ok) {
+            setSubmitted(true)
```
Comment (prophetsfall) [inline src/components/SurrenderForm.js]: clever!

File: src/components/SurrenderForm.js [hunk 5]
```diff
@@ -73,113 +90,123 @@ const SurrenderForm = props => {
...
-  )
+        <input type="submit" className="button" value="submit" />
+      </form>
+    )
+  } else {
+    return (
+      <>
+        <h3 id="surrender-review">Your application is pending review.</h3>
+        {setTimeout(() => setToHome(true), 5000)}
+        {toHome ? <Redirect to="/" /> : null}
```
Comment (prophetsfall) [inline src/components/SurrenderForm.js]: Love it!

File: public/style.css (hunk 1 above)
Comment (xshigeox) [reply public/style.css]: removed merge markers from css files

File: server/server.js (hunk 2 above)
Comment (xshigeox) [reply server/server.js]: readjusted to just take in petType.  were originally thinking if a user entered it incorrectly in the url when trying to get to the specific page

File: src/components/App.js (hunk 3 above)
Comment (xshigeox) [reply src/components/App.js]: decided to not use semi-colons.  Everyone has updated their prettier settings to not add them.

Comment (prophetsfall) [reply src/components/App.js]: :D

**Pull request merged**